yang/frr-ospfd.yang

```diff
@@ -346,6 +346,12 @@ module frr-ospfd {
         "The reference bandwidth in terms of Mbits per second.";
     }
 
+    leaf use-arp {
+      type boolean;
+      default "true";
+      description "ARP for neighbor table entry";
```

Comment:
I think sentences should end with a dot (like the rest of the file), but I'll leave this for the more experienced YANG modelers to answer.

Reply:
Thanks for noticing that!